pass_import.py

```diff
@@ -833,6 +965,16 @@ def report(arg, msg, paths):
         paths.sort()
         for path in paths:
             msg.echo(os.path.join(arg.root, path))
+    if binaries:
```

Comment:
A binary import is an import like any other, so you don't have to report it differently.